src-electron/sdk/matter.js

```diff
@@ -163,19 +155,21 @@ async function computeStorageImport(
   forcedExternal,
   attributeName
 ) {
-  if (
-    forcedExternal.byName &&
-    forcedExternal.byName[clusterName] &&
-    forcedExternal.byName[clusterName].includes(attributeName)
-  ) {
-    storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
-  }
+  forcedExternal.map((option) => {
```

Comment:
Is .map the right function to be used here? wondering if you just need forEach here since you just need to set storage policy

Reply:
fixed